crates/ironrdp-cliprdr/src/backend/mod.rs

```diff
@@ -0,0 +1,140 @@
+//! This module provides infrastructure for implementing OS-specific clipboard backend.
```

Comment:
Looks like there is no sub module anymore, so I suggest renaming `backend/mod.rs` to `backend.rs`